feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.service.spec.ts

```diff
@@ -124,4 +233,240 @@ describe('ConfiguratorCartEntryBundleInfoService', () => {
     expect(lineItems[2]).toEqual(expectedLineItem3);
     expect(lineItems[3]).toEqual(expectedLineItem4);
   });
+
+  describe('for enhanced OCC API format: ', () => {
+    it('should retrieve an existing lineitem for requested lineitemNumber from lineitemMap', () => {
+      const lineItemMap: Map<number, LineItem> = new Map();
+      const lineItemNumber: number = 123;
```

Comment:
```suggestion
      const lineItemNumber  = 123;
```
Similar for other declarations. Don't specify type if it can be derived from the value.